src/main/java/duke/exceptions/DukeException.java

```diff
@@ -8,4 +10,9 @@
     public DukeException(String message) {
         super("Oops! " + message);
     }
+
+    public DukeException(Class className, String message) {
+        super("Oops! " + message);
+        Log4jLogger.logException(className,message);
+    }
```

Comment:
Nice, we can then slowly shift all our remaining exception to this DukeExceptionWithLogger